lessons/static/css/commonlesson.css

```diff
@@ -395,7 +395,7 @@ html, body{
 }
 
 .tiplink-assessment a:link, .tiplink-assessment a:visited {
-    color: #0094ca;
+    color: #7665a0;
```

Comment:
are these styles currently used?

Reply:
yes

Reply:
can we get some before/after screenshots of the components that use them? I'd love to see what effects this change will have on existing content.

Reply:
This line was added when I added the mini-rubric display in code studio pull through a couple of weeks back but not used yet at that time since in the code studio pull through we were not using the left side icon.  You can see the icon this styles in the "Result" view above. The circle with a check mark next to Tag Poster is purple as a result of this change. Without this change, it would be blue. I added before and after screenshots where appropriate above

Reply:
Excellent! Thank you